blocks/url-input/test/button.js

```diff
@@ -0,0 +1,69 @@
+/**
+ * External dependencies
+ */
+import { shallow, mount } from 'enzyme';
+
+/**
+ * External dependencies
+ */
+
+/**
+ * Internal dependencies
+ */
+import UrlInputButton from '../button';
+
+describe( 'UrlInputButton', () => {
+	it( 'should has valid class in wrapper tag', () => {
+		const wrapper = shallow( <UrlInputButton /> );
+		expect( wrapper.hasClass( 'blocks-url-input__button' ) ).toBeTruthy();
```

Comment:
When validating boolean values `toBe( true )` seems to be a better choice, which performs the comparison using `===`. See [this thread on StackOverflow](https://stackoverflow.com/a/32767435/4791119) where it is greatly explained.